javatests/arcs/core/data/expression/TypeEvaluatorTest.kt

```diff
@@ -32,10 +32,10 @@ import org.junit.Test
 import org.junit.runner.RunWith
 import org.junit.runners.JUnit4
 
-/** Tests for [TypeEvaluator]. */
+/** Tests for [TypeEvaluator]. This was previously named [TypeCheckerTest]. */
```

Comment:
You don't need to say this, people will be able to figure it out. In fact, now that the test file has the same name as the class under test, you don't need to have a comment here at all.